cluster/addons/fluentd-elasticsearch/kibana-image/Makefile

```diff
@@ -14,7 +14,7 @@
 
 .PHONY:	build push
 
-TAG = v4.6.1
+TAG = v4.6.1-1
```

Comment:
yeah, it appears to correlate with the Kibana version - https://github.com/ixdy/kubernetes/blob/1c2ea2808012dcea8e785f14c0e155ed66c2808a/cluster/addons/fluentd-elasticsearch/kibana-image/Dockerfile#L22

Alternately, there is a Kibana 4.6.3 we could upgrade to, along with updating the base image. (It's actually up to 5.1.1, but 5.0 had some breaking changes which may or may not affect us.)